test/AutoRemediateS3-002.test.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+const source = require('../functions/AutoRemediateS3-002.js');
```

Comment:
`const source = require('../functions/AutoRemediateS3-002')`

https://medium.freecodecamp.org/requiring-modules-in-node-js-everything-you-need-to-know-e7fbd119be8